update-crt.py

```diff
@@ -37,7 +37,7 @@ def main():
     if args.update_samples == True:
         update_samples()
     elif args.update_sdk_text == True:
-        sdk_version = get_latest_github_version("https://github.com/aws/aws-iot-device-sdk-java-v2.git")
+        sdk_version = args.version
```

Comment:
It would be nice if we both set a version number by args or directly pull from github.
Something like:
```
if (args.version): 
        sdk_version = args.version
else:
        sdk_version = get_latest_github_version("https://github.com/aws/aws-iot-device-sdk-java-v2.git")
```

Reply:
if a version isn't provided, it's automatically set to the github version on line 28

Reply:
It's `aws-crt-java` repo on line 28, so `sdk_version` will be wrong in case when `args.version` is None